Makefile

```diff
@@ -32,6 +32,12 @@ docker-playground-build:
 	# This method does NOT require that you have Go, NPM, etc. installed locally.
 	DOCKER_BUILDKIT=1 docker build -f install/docker/Dockerfile -t layer5/meshery --build-arg TOKEN=$(GLOBAL_TOKEN) --build-arg GIT_COMMITSHA=$(GIT_COMMITSHA) --build-arg GIT_VERSION=$(GIT_VERSION) --build-arg RELEASE_CHANNEL=${RELEASE_CHANNEL} --build-arg PROVIDER=$(LOCAL_PROVIDER) --build-arg PROVIDER_BASE_URLS=$(MESHERY_CLOUD_PROD) .
 
+## Build Meshery Server and UI container for e2e testing.
+docker-build-testing-env:
```

Comment:
Good! I wonder if we don't want to include these two new targets in the `.PHONY:` line so that the targets are readily invocable.

Reply:
Sure